openmc/region.py

```diff
@@ -46,6 +46,25 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def get_surfaces_from_region(self, surfaces = OrderedDict()):
```

Comment:
Remove the whitespace around the `=` for the keyword argument `surfaces`. I *think* no whitespace is PEP8 for these circumstances, though even if not, this would match the convention used elsewhere in the PyAPI.